### PR TITLE
Lean integration (merge strategies)

### PR DESCRIPTION
**Pull request by kommander**
Preparation for 0.3.0 release

- [x] Implement simple merge strategy for steffi-graph

### CODE REVIEW

File: src/load-schema.js [hunk 1]
```diff
@@ -37,65 +57,59 @@ const loadSchema = async ({ modulePaths, mockMode, useFileSchema, filterSubscrip
...
-    .filter(config => config.dependencies)
-    .forEach(config => config.dependencies
-      .forEach(dependency => {
-        if (!sourceNames.includes(dependency)) {
-          throw new Error(`Cannot load module "${config.name}", ` +
-            `because missing dependency "${dependency}"`);
-        }
-      })
-    );
+  const moduleConfigMap = configs.reduce((prev, curr) => Object.assign(prev, {
```
Comment (TimSusa) [inline src/load-schema.js]: const moduleConfigMap = configs.reduce((prev, cur) => ({...prev, [cur.name]: cur}), {}); ?

**Comment by TimSusa** [comment]
Wow, I had a look to the diff, a hell lot of work, man. Let me know, if I can help. Anyhow, it looks very good to me.

**Comment by kommander** [comment]
Thanks. @TimSusa @MikeBild What's missing for steffi? Handing through directives?

File: src/load-schema.js (hunk 1 above)
Comment (kommander) [reply src/load-schema.js]: Same same. Feel free to change it.

**Comment by kommander** [comment]
Lean has been deployed with beta.5, merging this, keeping the beta status for now until red delivery has been deployed.

**Pull request merged**